EPRN.PRNS.API/appsettings.json

```diff
@@ -5,5 +5,10 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "AppSettings": {
+    "DeductionAmount_Exporter": 0.15,
+    "DeductionAmount_Reprocessor": 0.16,
+    "DeductionAmount_ExporterAndReprocessor": 0.17
+  }
```

Comment:
Why are there app settings in PRN api, yet no other changes? Are these required?

Reply:
Removed.